content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
+Cheriton School of Computer Science. For the remainder of this document,
+this standing committee will be referred to as the "WiCS Committee". All
+references to: "the undergraduate committee", "the committee", and
+"committee members" refer to the Undergraduate Subcommittee unless
+otherwise specified.
+
+## Purpose and Vision ##
+
+The undergraduate committee aims to create a safe environment for cis
+and trans women as well as non-binary individuals in computer science,
```

Comment:
I'd change "as well as" to "and", because otherwise enbies seem to be relegated to second-class status here. If we want to be consistent with our big CSters branding, we could use cis & trans so there aren't too many "and"s.